index.js

```diff
@@ -1,18 +1,151 @@
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+// Valid string inputs ---------------------------------------------
+const correctAnswer = ['yes', 'y', 'ys'];
+const wrongAnswer = ['no', 'n'];
+const highAnswer = ['higher', 'h', 'high'];
+const lowAnswer = ['lower', 'l', 'low'];
+const playComputerGuessing = ['computer', 'comp', 'c'];
+const playPlayerGuessing = ['player', 'play', 'p'];
+const validGameTypeInput = [...playComputerGuessing, ...playPlayerGuessing];
+let setRangeMax = process.argv[3] || 100;
+let setRangeMin = process.argv[2] || 1;
+
+
+// Function list ---------------------------------------------------
+
+// readline function
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
-start();
+// Sanatizes string input
+function sanitizeString(string) {
+  string = string
```

Comment:
You're going to want to use the `let` keyword when you define string to avoid creating a global variable.